app/controllers/tags_controller.rb

```diff
@@ -143,6 +143,8 @@ def list
   Discourse.filters.each do |filter|
     define_method("show_#{filter}") do
       @tag_id = params[:tag_id].force_encoding("UTF-8")
+      fetch_tag
```

Comment:
I tried updating the `before_action :fetch_tag` list to include `*show_methods`, but that didn't seem to work properly?